src/Groups/libraries/primitivegroups.jl

```diff
@@ -0,0 +1,125 @@
+export
+    all_primitive_groups,
+    blocks,
+    isprimitive,
+    maximal_blocks,
+    number_primitive_groups,
+    primitive_group,
+    representatives_minimal_blocks
```

Comment:
Move to `gsets.jl`?

Reply:
Ok, I'll do it when I will create the file `gsets.jl`.